scss/mixins/_hover.scss

```diff
@@ -1,14 +1,15 @@
 @mixin hover {
-  @if $enable-hover-media-query {
-    // See Media Queries Level 4: http://drafts.csswg.org/mediaqueries/#hover
-    // Currently shimmed by https://github.com/twbs/mq4-hover-shim
-    @media (hover: hover) {
-      &:hover { @content }
-    }
-  }
-  @else {
+  // TODO: re-enable along with mq4-hover-shim
```

Comment:
Separate PR. Would need to first change mq4-hover-shim to implement PostCSS's standard plugin API.
Also depends on the answer to https://github.com/twbs/bootstrap/issues/19968#issuecomment-227212987